index.js

```diff
@@ -6,30 +6,17 @@ var xtend = require("xtend")
 
 module.exports = createXHR
 createXHR.XMLHttpRequest = window.XMLHttpRequest || noop
-createXHR.XDomainRequest = "withCredentials" in (new createXHR.XMLHttpRequest()) ? createXHR.XMLHttpRequest : window.XDomainRequest
 createXHR.qsSerialize = null // Define this as a function to support the `qs` option
 
-forEachArray(["get", "put", "post", "patch", "head", "delete"], function(method) {
+;
```

Comment:
You could always toss `var methods = ["get","put","post","patch","head","delete"]` at the top of the file.